src/libUtils/EvmUtils.cpp

```diff
@@ -98,3 +102,59 @@ Json::Value EvmUtils::GetCallContractJson(const RunnerDetails& details) {
 
   return arr_ret;
 }
+
+bool EvmUtils::EvmUpdateContractStateAndAccount(
```

Comment:
This is clean and nice, thank you!

Looking forward to seeing account deletion and handling reset_storage.  Cannot really test without reset_storage - I expect it could be frequent.

Reply:
Account deletion is being looked at by @KaustubhShamshery , handle reset we are working on right now, doesn't exist in current usage, need to make sure it travels full depth through the objects temps,cached etc.